cmake/Modules/FindMETIS.cmake

```diff
@@ -45,16 +45,24 @@ if (METIS_INCLUDE_DIR)
     string(REGEX REPLACE "^#define[\t ]+METIS_VER_MINOR[\t ]+([0-9]+).*" "\\1" METIS_VERSION_MINOR "${metis_version_str_minor}")
     set(METIS_VERSION ${METIS_VERSION_MAJOR}.${METIS_VERSION_MINOR})
     find_library(METIS_LIBRARY ${METIS_LIB_NAME} HINTS ${METIS_DIR} ENV METIS_DIR PATH_SUFFIXES lib lib64)
+    if (METIS_VERSION VERSION_GREATER_EQUAL 5.0)
+        find_library(METIS_GKLIB_LIBRARY GKlib HINTS ${METIS_DIR} ENV METIS_DIR PATH_SUFFIXES lib lib64)
+    else()
+        # set it to a non-empty value to satisfy find_package_handle_standard_args
+        set(METIS_GKLIB_LIBRARY libGKlib.so)
```

Comment:
I assume `libGklib.so` is the main library from METIS and it always exists (or it will be checked through `find_package_handle_standard_args`)

Reply:
no, this is just a simple way to avoid making the arguments of `find_package_handle_standard_args` conditional.

Reply:
could you elaborate it more? if you put libGklib.so which is not existed, will find_package_handle_standard_args try to look for it or put it as interface linking library?